toolkit/tools/imagecustomizer/container/test-mic-container.sh

```diff
@@ -0,0 +1,36 @@
+# Test container by running run.sh script inside it.
+set -eux
+
+containerTag=$1
+
+outputImage=$(readlink -f "../../out/containertestoutput.vhdx")
```

Comment:
Place `SCRIPT_DIR=$(dirname "${BASH_SOURCE[0]}")` at the top of the file and then use `SCRIPT_DIR` when referencing files relative to this script. This avoids this script depending on the current working directory being a particular value.

Reply:
good to know, thanks!!